pkg/cmd/root/extension_test.go

```diff
@@ -16,6 +17,7 @@ func TestNewCmdExtension_Updates(t *testing.T) {
 	tests := []struct {
 		name               string
 		extCurrentVersion  string
+		extFullName        string
```

Comment:
@andyfeller - this should be removed because this is not being used by the tests.